bin/autorun.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+## this is the default autorun script
+## file should run automatically after init.sh completes.  
+## this script downloads your configured run.env, system.env and any custom container requests
+
+INSTALL_DIR_TEMP="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null 2>&1 && pwd )"
+
+## retrieve the bucket name you sent the instance earlier
+BUCKET=$(cat $INSTALL_DIR_TEMP/bucket.txt)
+
+## load some variables
+source $INSTALL_DIR_TEMP/bin/scripts_wrapper.sh
+source $INSTALL_DIR_TEMP/bin/activate.sh
+
+
+## get the updatated run.env and system.env files you created and stashed in s3
+aws s3 cp s3://$BUCKET/run.env $INSTALL_DIR_TEMP/run.env
+aws s3 cp s3://$BUCKET/system.env $INSTALL_DIR_TEMP/system.env
+
+
+## get the right docker containers, if needed
+SYSENV="$INSTALL_DIR_TEMP/system.env"
+SAGEMAKER_IMAGE=$(cat $SYSENV | grep DR_SAGEMAKER_IMAGE | sed 's/.*=//')
+ROBOMAKER_IMAGE=$(cat $SYSENV | grep DR_ROBOMAKER_IMAGE | sed 's/.*=//')
+
+docker pull awsdeepracercommunity/deepracer-sagemaker:$SAGEMAKER_IMAGE
+docker pull awsdeepracercommunity/deepracer-robomaker:$ROBOMAKER_IMAGE
+
+dr-update
+
+date | tee $INSTALL_DIR_TEMP/DONE-AUTORUN
+
+## start training -- commented out for now until I can figure out how to do it properly
+cd $INSTALL_DIR_TEMP/scripts/training 
+nohup ./start.sh &
```

Comment:
nohup and & is not needed; everything that remains are the containers that are anywaeys preserved.

Reply:
done